### PR TITLE
Added check if repo has ending slash

### PR DESCRIPTION
**Pull request by yvesh**
If the repo setting in the acceptance.suite.yml does not have an ending slash it fails.. This is pretty annoying. Added a check for that.

### CODE REVIEW

**Comment by therunnergit** [comment]
What about
    
    $path = rtrim($path, "/") . "/";

**Comment by puneet0191** [comment]
Nice Nice!

**Comment by Kubik-Rubik** [comment]
@yvesh Could you update your PR with @therunnergit's suggestion?

File: src/JoomlaBrowser.php [hunk 1]
```diff
@@ -335,6 +335,13 @@ public function installExtensionFromFolder($path, $type = 'Extension')
 		$I->waitForText('Extensions: Install','30', ['css' => 'H1']);
 		$I->click(['link' => 'Install from Folder']);
 		$this->debug('I enter the Path');
+
+		//Make sure path has an ending slash
+		if (substr($path, -1) != "/")
+		{
+			$path = $path . "/";
+		}
```
Comment (javigomez) [inline src/JoomlaBrowser.php]: :+1:  nice :smile:

**Comment by yvesh** [comment]
Done. Thanks @therunnergit

**Comment by Kubik-Rubik** [comment]
Thank you @yvesh and testers! Merged.

**Pull request merged**